libpcsxcore/psxmem.c

```diff
@@ -389,6 +389,10 @@ void psxMemWrite32(u32 mem, u32 value) {
 						memset(psxMemWLUT + 0x0000, 0, 0x80 * sizeof(void *));
 						memset(psxMemWLUT + 0x8000, 0, 0x80 * sizeof(void *));
 						memset(psxMemWLUT + 0xa000, 0, 0x80 * sizeof(void *));
+#ifdef ICACHE_EMULATION
+						/* Required for icache interpreter otherwise Armored Core won't boot on icache interpreter */
+						psxCpu->Reset();
+#endif
```

Comment:
I think Notify(R3000ACPU_NOTIFY_CACHE_ISOLATED) makes more sense here, as writes to 0xfffe0130 control the cache. I think it was here in the older patch?

Reply:
Yeah it was in my older patch but i took it out due to the mem_bak changes. I brought it back and the other one now.